### ISSUE TITLE
how to use clustergrammer2 with "dash" for my web app

### ISSUE THREAD

**Issue by jessielyons** [ismms-himc/clustergrammer2#124]
Hello,

I am quite lost on how to use clustergrammer2 with "dash". I really want to show a heatmap in my app and appreciate your feedback asap. 

My current clustergrammer2 code which works.

num_rows = 71
n1 = Network(CGM2)

n1.load_df(np.log2(df_allgrp_fill_transposed), meta_col=df_allgrp_fill_meta, col_cats=col_cats)
n1.set_global_cat_colors(df_colors)
n1.filter_N_top(axis='row', N_top=num_rows, rank_type='var')
n1.set_manual_category(col='Tissue', preferred_cats=df_colors)
n1.widget()

Thanks so much!

**Comment by jessielyons** [comment]
Hey, anyone any response? So I can atleast create a .json file. But how do I read this json file via dash to build a dashboard?

Please help :)

**Comment by cornhundred** [comment]
Hi, I haven't used Dash but I can try to look into it.

**Comment by jessielyons** [comment]
Hi @cornhundred ,

Thanks and look forward to your help. Alternatively, do you have a script available on how to display the .json file perhaps into a website for this heatmap or an .html file? Thanks again!

**Comment by cornhundred** [comment]
Yes, @jessielyons this ObservableHQ collection has several examples of how the visualization can be embedded into a JavaScript application https://observablehq.com/collection/@cornhundred/clustergrammer-gl Let me know if you have any questions
